src/Auth/HasIdentity.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+use Tempest\Database\IsModel;
+
+trait HasIdentity
+{
+    use IsModel;
+
+    protected const string IDENTIFIER = 'email';
+
+    protected const string SECRET = 'password';
+
+    public function source(): string
+    {
+        return (string) static::table();
+    }
+
+    public function identifier(): string
+    {
+        return static::IDENTIFIER;
```

Comment:
Oh, we already have a method for it. Let's then get rid of the constant alltogether.